### DISCUSSION TITLE
¿Cómo puedo agregar etiquetas personalizadas a puntos?

### DISCUSSION THREAD

**Discussion by AlonsoLl** [qgispe/discusiones#18] [Preguntas&Respuestas] [answered]
Buen día comunidad de QGIS Perú, quisiera que me puedan apoyar a como poder realizar un etiquetado personalizado. Como pueden observar en la primera captura se visualiza mi mapa con los diferentes cortes (polígonos) y en la segunda imagen ya se muestran mis puntos y quisiera que cada punto dependiendo al corte donde esta ubicado se muestre de diferente color para poder diferenciar. Espero me puedan ayudar indicando como, muchas gracias.

![image](https://user-images.githubusercontent.com/50957145/126331336-e2358984-33a3-4371-85d9-8b9860e6b0ab.png)

![image](https://user-images.githubusercontent.com/50957145/126331289-59874107-7d7c-4056-89a0-5751e5094024.png)

**Comment by ambarja** [comment, marked as answer]
Hola @AlonsoLl  creo que lo más adecuado es hacer una interpolación de puntos para poder plasmar la información que deseas trasmitir, pero a tu pregunta de como personalizar etiquetas de acuerdo a los límites tendrías que usar ![qgis](https://user-images.githubusercontent.com/23284899/126374123-4fea129a-9254-4f45-b5a2-6c8f86bdfe4b.png) por
